app/lib/common/widgets/chat/convo_with_avatar_card.dart

```diff
@@ -70,7 +68,14 @@ class ConvoWithAvatarInfoCard extends ConsumerWidget {
                 onFocusChange: onFocusChange,
                 onLongPress: onLongPress,
                 leading: avatarWithIndicator(context, ref),
-                title: buildDisplayName(context, ref),
+                title: Text(
+                  avatarInfo.displayName ?? roomId,
+                  style: Theme.of(context)
```

Comment:
When given a roomInfo we'd unfortunately try to fetch a chat name for an object we don't have a local representation for. Thus failing and just showing `roomId`s. Instead we are now using the given `AvatarInfo` that is pushed in and always has the resolved displayName regardless of the origin object type.